### ISSUE TITLE
Bug : Tensorboard step in loss and accuracy plots shifted by 1 to the left on retraining 

### ISSUE THREAD

**Issue by kamal4493** [keras-team/keras#9318]
Hi 

I found a bug in Keras Tensorboard. 
If I train a model and use tensorboard to log metrics for a fixed number of epochs , everything works fine.
But if I retrain the model from the last epoch, the validation and accuracy plots gets shifted by 1 to the left.
Example

I train a model for 50 epochs. Right now everything looks good in tensorboard. If I retrain the same model with saved weights and take initial epoch from last finished epoch, tensorboard start plotting values from step 50 instead of step 51. so epoch 51 values are plottted on step 50.

Tell me if I am doing something wrong.Although I made sure to double check my code
